main.js

```diff
@@ -74,5 +79,9 @@ function placeHours(hours, minHours, maxHours, lineStartX, lineEndX){
 $(document).ready(function(){
   drawNumberline();
 
+  $('input.hours_input').on('change', function(){
```

Comment:
So, truth be told, the best way to do this is to have the input be in an actual form - then HTML5 validations will make sure that the `step` value of the input is being adhered to.